akamai/data/stage/base_rules.json

```diff
@@ -424,7 +424,7 @@
                                         "options": {
                                             "matchOperator": "MATCHES_ONE_OF",
                                             "values": [
-                                                "/beta/rhel/vulnerability*"
+                                                "/beta/openshift/subscriptions"
```

Comment:
This redirect will only work for the exact path `/beta/openshift/subscriptions`. Is that intentional?

Reply:
Same here, but I will confirm with Nicole and Ryan.

Reply:
Updated the link with *, making the redirect rule more generic seems better @ryelo